src/shared/firewall-util-nft.c

```diff
@@ -807,20 +807,26 @@ int fw_nftables_init(FirewallContext *ctx) {
         if (r < 0)
                 return r;
 
-        r = fw_nftables_init_family(nfnl, AF_INET);
-        if (r < 0)
-                return r;
-
-        if (socket_ipv6_is_supported()) {
-                r = fw_nftables_init_family(nfnl, AF_INET6);
+        if (init_tables) {
```

Comment:
Maybe, it is better to split `fw_nftables_init_family()` completely from `fw_nftables_init()`, and call `_init_family()` when necessary.

Reply:
ok